soc/renesas/ra/ra4m1/Kconfig

```diff
@@ -9,3 +9,26 @@ config SOC_SERIES_RA4M1
 	select DYNAMIC_INTERRUPTS
 	select TIMER_READS_ITS_FREQUENCY_AT_RUNTIME
 	select XIP
+
+config SOC_OPTION_SETTING_MEMORY
+	bool "Option Setting Memory"
+	default y
+
+config SOC_OPTION_SETTING_LVDAS_DISABLE
+	bool "Disable voltage monitor 0 after reset"
+	default y
+	depends on SOC_OPTION_SETTING_MEMORY
+
+config SOC_OPTION_SETTING_HOCO_DISABLE
+	bool "Disable HOCO oscillator after reset"
+	depends on SOC_OPTION_SETTING_MEMORY
+
+config SOC_OPTION_SETTING_HOCO_FREQ
+	int "HOCO oscillator frequency"
+	default $(dt_node_int_prop_int,/clocks/hoco,clock-frequency)
+	depends on SOC_OPTION_SETTING_MEMORY
+
+config SOC_OPTION_SETTING_VDSEL
```

Comment:
I think it's better to create `dts/bindings/hwinfo/renesas,ra-lvd.yaml` and define it in the devicetree node.

Reply:
I don't think this setting needs to be done in this PR.
Since the hwinfo driver is used to obtain the startup factor after a reset, I think it would be a good idea to create a devicetree definition related to this and use that.
(If you are new to Zephyr development, creating a hwinfo driver brings good experience.)

Reply:
@soburi, Agreed, I would like to implement this setting via the device tree but do it in another PR. Shall I leave the VDSEL as it is for this PR or maybe remove the option?

Reply:
Don't leave options that don't work, as they confuse users.